man/systemd.netdev.xml

```diff
@@ -571,6 +571,16 @@
           <xi:include href="version-info.xml" xpointer="v248"/>
         </listitem>
       </varlistentry>
+     <varlistentry>
+        <term><varname>BroadcastThresholdLimit=</varname></term>
+        <listitem>
+          <para>Specifies the broadcast queueing threshold on macvlan devices. This controls which multicast
+          packets will be processed in a workqueue instead of inline. Setting this to -1 disables broadcast
+          queueing altogether An integer in the range -1…2147483646. Defaults to unset.</para>
```

Comment:
Missing period after `altogether`. Also, `2147483647` seems to be allowed.

Reply:
No I tested it's allowed till `2147483646`

Reply:
```-2147483648``` to ```2147483646``` is valid